vlib/x/json2/json_module_compatibility_test/json_todo_test.vv

```diff
@@ -69,22 +69,7 @@ struct SomeGame {
 
 //! BUGFIX - .from_json(res)
 fn test_encode_decode_sumtype() {
-	t := time.now()
-	game := SomeGame{
-		title: 'Super Mega Game'
-		player: Human{'Monke'}
-		other: [
-			Entity(Item{'Pen'}),
-			Item{'Cookie'},
-			Animal.cat,
-			'Stool',
-			t,
-		]
-	}
-
-	enc := json.encode(game)
-
-	assert enc == '{"title":"Super Mega Game","player":{"name":"Monke","_type":"Human"},"other":[{"tag":"Pen","_type":"Item"},{"tag":"Cookie","_type":"Item"},1,"Stool",{"_type":"Time","value":${t.unix_time()}}]}'
+	enc := '{"title":"Super Mega Game","player":{"name":"Monke","_type":"Human"},"other":[{"tag":"Pen","_type":"Item"},{"tag":"Cookie","_type":"Item"},1,"Stool",{"_type":"Time","value":${t.unix_time()}}]}'
```

Comment:
No, this should stay. The goal of the is very explicitly (from the name), to test a round trip (encoding, then decoding), and checking that the input and output match.

Just deleting the first part, defeats the test's purpose.

Reply:
It is a `.vv` file. I remove from there to put in `.v` file

Reply:
oh, ok then.